.travis.yml

```diff
@@ -26,7 +26,13 @@ install:
   - PATH=$HOME/gopath/bin:./third_party/etcd:$PATH ./hack/verify-gendocs.sh
 
 before_script:
-  - npm install karma karma-junit-reporter karma-phantomjs-launcher karma-jasmine
+  - travis_retry npm install karma@~0.10 karma-phantomjs-launcher@~0.1.4 karma-cli@0.0.4 karma-jasmine@^0.1.5 karma-junit-reporter@^0.2.2 karma-story-reporter@^0.3.1 protractor@^2.1.0
```

Comment:
Can you package this stuff up in a script and call that? That'd be much clearer (and easier to try locally if desired).